### ISSUE TITLE
Any updates?

### ISSUE THREAD

**Issue by NVader2000** [zsarnett/Custom-Grid-View#26]
**Is your feature request related to a problem? Please describe.**
Yes and no. I am running 2021.7.4 and neither install method seems to work. It is possible that I am missing something, but it appears to straight forward.

**Describe the solution you'd like**
Further development. I understand that some elements are not friendly on all screens. Hence why I seek the Drag-n-Drop and resize option. It is ludicrous that HA does not include these functions natively in both the web portal and more importantly on mobile and tablet devices. 

**Describe alternatives you've considered**
N/A

**Additional context**

**Comment by zsarnett** [comment]
This may not be updated to the latest - I need to look at the recent changes and update the code

**Comment by NVader2000** [comment]
Hi Zack if you could please (begging). I hope someday that this will become a native function in HA. However, I foresee that there will be a need to have separation of some sort for each device that is to render the Loverlace UI. So, a way to have multiple dashboards based on devices that are running the HA app.

**Comment by Misiu** [comment]
I'm really looking forward to seeing this as a default layout in HA.
It feels much more intuitive than the Mansory layout.
Not sure if every card supports resizing and how should the layout behave on different screen sizes (responsive view), but I'm sure you will figure something out :)
Thank you Zack for awesome work and hopefully, you will return soon to HA development 💪

**Comment by NVader2000** [comment]
Not sure that HA will have any control over the layout on different devices. Maybe through the device identity where it could read the screen resolution and size? However, since there is a HA app for iOS/Android this might depend on the app developer getting this functionality implemented. Thoughts?

**Comment by ScottG489** [comment]
@zsarnett What happened with this project? I'm really surprised this isn't even the default method of organizing dashboard cards. The up/down buttons feel archaic.

**Comment by ScottG489** [comment]
@pannal That portrait/landscape issue I experience also. Do you have problems with the layout not saving?

**Comment by zsarnett** [comment]
No Updates. There will be plans to have something to this extent in the future. I do not know when

**Comment by Misiu** [comment]
I also hope for the continuation of this awesome project.
Having this build-in into HA would be a game-changer!

**Comment by danlun** [comment]
Yes, after been struggling to get horizontal, vertical and grid layout done, you just wonder why this isn’t the default way to do dashboard layout. Please make more on this and the default way in HA. Great job!
